game/index.html

```diff
@@ -7,9 +7,12 @@
   </head>
   <body>
 
-<canvas id="gamecanvas" width="1000" height="500">
+<canvas id="gamecanvas1" style="z-index: 1;" width="1000" height="500">
+<!-- <canvas id="gamecanvas2" style="z-index: 2;" width="1000" height="500"> -->
```

Comment:
Think you can leave out this second canvas. Avoid using numeric counters in names. Name things for their actual meaning. Eg. `<canvas id="background'>` and `<canvas id="foreground'>`.